src/main/scala/aspects/RenameModulesAspect.scala

```diff
@@ -1,46 +0,0 @@
-// See LICENSE.SiFive for license details.
```

Comment:
circt only support prefix module name, see [here](https://circt.llvm.org/docs/Passes/#-firrtl-prefix-modules-prefixes-names-of-modules-and-mems-in-a-hierarchy)
But I do like this feature, hopefully chisel can provide it in the chisel5